.docker/entrypoint/sh/entrypoint.sh

```diff
@@ -1,15 +1,42 @@
-#!/bin/bash
-set -o errexit # Immediately abort if any command fails
+#!/usr/bin/env bash
+set -o errexit # Abort if any command fails
 
-[ "${DEBUG:-false}" = "true" ] && set -x
+parse_args() {
+    if [[ "$@" =~ \-\-env[\=\ ]+([^\ ]+) ]]; then
+        env="${BASH_REMATCH[1]}"
+        [ ${DEBUG:-false} ] && echo "Detected --env='$env'."
+    fi
+}
 
-ruby "${JEKYLL_VAR_DIR}/entrypoint/lib/gemfile_generator_exec.rb"
+# echo expanded commands as they are executed (for debugging)
+enable_expanded_output() {
+    if [ ${DEBUG:-false} ]; then
+        set -o xtrace
+        set +o verbose
+    fi
+}
 
-default_gemfile="${JEKYLL_DATA_DIR}/Gemfile.generated"
+main() {
+    parse_args "$@"
 
-bundle check --gemfile="$default_gemfile" \
-  || bundle install --gemfile="$default_gemfile"
+    enable_expanded_output
 
-bundle config set gemfile "$default_gemfile"
+    ruby "${JEKYLL_VAR_DIR}/entrypoint/lib/gemfile_generator_exec.rb"
+
+    default_gemfile="${JEKYLL_DATA_DIR}/Gemfile.generated"
+
+    bundle check --gemfile="$default_gemfile" \
+      || bundle install --gemfile="$default_gemfile"
+
+    if [[ -n $env ]]; then
+        [ ${DEBUG:-false} ] && echo "Exporting JEKYLL_ENV='$env'."
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)